services/src/unit-test/java/uk/gov/moj/sdt/services/utils/GenericXmlParserTest.java

```diff
@@ -33,26 +33,35 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
 
 import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 import uk.gov.moj.sdt.utils.SdtContext;
 import uk.gov.moj.sdt.utils.Utilities;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 /**
  * Test class for the GenericXmlParser.
  *
  * @author d276205
  */
-public class GenericXmlParserTest extends AbstractSdtUnitTestBase {
-    /**
-     * Logger object.
-     */
-    private static final Logger LOGGER = LoggerFactory.getLogger(GenericXmlParserTest.class);
+@ExtendWith(MockitoExtension.class)
+class GenericXmlParserTest extends AbstractSdtUnitTestBase {
+
+    private static final String URL_TARGET_APP = "http://ws.sdt.moj.gov.uk/2013/sdt/targetApp";
+    private static final String URL_BULK_FEEDBACK_RESPONSE_SCHEMA =
+            "http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema";
+    private static final String URL_INDV_RESPONSE_SCHEMA = URL_TARGET_APP + "/IndvResponseSchema";
+    private static final String URL_SUBMIT_QUERY_RESPONSE_SCHEMA = URL_TARGET_APP + "/SubmitQueryResponseSchema";
+    private static final String FAILED_TO_FIND_EXPECTED_RESPONSE
+            = "Failed to find expected response";
+    private static final String TARGET_APP_DETAIL = "targetAppDetail";
+
+    private static final String SRC_UNIT_TEST_RESOURCES = "src/unit-test/resources/";

```

Comment:
Change constant name so it better reflects what it is.  Something like UNIT_TEST_RESOURCES_PATH perhaps.

Reply:
renamed

Reply:
Confirmed constant has been renamed.